_layouts/job_post.html

```diff
@@ -3,43 +3,47 @@
 title_prefix: "Jobs with Compiler:"
 ---
 
-<h1>{{ page.title }}{% if page.type %} ({{ page.type }}){% endif %}</h1>
+<div class="row">
+  <div class="offset-md-2 col-md-7 col-12 mt-5 pt-5 mb-5 pb-5">
```

Comment:
> mt-5 pt-5 mb-5 pb-5

Could this be simplified by saying `my-5 py-5`?

Reply:
Yea it could. Eventually though, I think the margins should be set on the top nav. 
<img width="723" alt="image" src="https://github.com/compilerla/compiler.la/assets/3673236/e57add84-06f6-4306-b21a-4ded3806207a">
<img width="208" alt="image" src="https://github.com/compilerla/compiler.la/assets/3673236/c48d3f15-207e-403f-b87d-f80075f87b70">

Will do this in my next PR!

Reply:
Ok, sounds good